<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>567</width>
    <height>427</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>567</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>521</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#botaoEntrar{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#botaoEntrar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#botaoEntrar:hover{
	background-color: rgb(247, 139, 31);
}</string>
    </property>
    <widget class="QLabel" name="telaInput">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>261</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255,255,255,255);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="telaLogo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>271</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0  rgb(247, 139, 31), stop:1 rgb(0, 85, 148));
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputUsuario">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,200);
color:rgb(0,0,0);
padding-bottom:7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Usu√°rio</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputSenha">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,200);
color:rgb(0,0,0);
padding-bottom:7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botaoEntrar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="logoMapaSala">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>200</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Courier&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Mapa de Sala&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="logoLogin">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0, 0, 0, 200);</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="logoSenac">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../VSCode testes/projeto integrador/logoSenac.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="respostaLogin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botaoSair">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>358</y>
       <width>65</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#botaoSair{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#botaoSair:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#botaoSair:hover{
	background-color: rgb(247, 139, 31);
}</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botaoSair</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
